src/Billing/Billing/Cmdlets/BillingPeriods/GetAzureRmBillingPeriod.cs

```diff
@@ -19,9 +19,11 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Management.Automation;
+using Microsoft.WindowsAzure.Commands.Common.CustomAttributes;
 
 namespace Microsoft.Azure.Commands.Billing.Cmdlets.BillingPeriods
 {
+    [CmdletDeprecation("2.0.0")]
```

Comment:
Please provide the migration instruction for the cmdlet here.

Reply:
I am not sure what migration instruction should contain. We do not have a comparable APIs for BillingPeriod and EnrollmentAccount cmdlets in new version of PowerShell since we currently  do not have plan to support it in our new version of APIs and SDKs. Is it possible to deprecate without providing migration instruction.